Dockerfile.base

```diff
@@ -54,20 +42,19 @@ ENV PLAYWRIGHT_SKIP_DOWNLOAD=true
 
 # Add user calypso with uid 1003, give it sudo permissions
 RUN apt-get update \
-	&& apt-get install -y sudo zip jq \
+	&& apt-get install -y sudo zip jq curl \
 	&& adduser --uid $UID --disabled-password $user \
 	&& echo "$user ALL=(ALL) NOPASSWD: ALL" > /etc/sudoers.d/$user \
 	&& chmod 0440 /etc/sudoers.d/$user \
 	&& chown $UID /calypso \
-	&& rm -rf /var/lib/apt/lists/*
+	&& rm -rf /var/lib/apt/lists/* \
+	&& apt autoremove --purge \
+	&& apt autoclean
```

Comment:
There are two camps for performing `apt` cleanup at end: those that prefer `autoclean` and those that prefer `clean`.

I have read that `clean` is more aggressive; does anyone have opinions either way?

Reply:
No opinions from me -- I think we should prioritize performance over anything else